cylc/flow/task_job_mgr.py

```diff
@@ -310,7 +309,7 @@ def submit_task_jobs(self, workflow, itasks, curve_auth,
                                 itask.tdef.rtconfig['platform'],
                                 bad_hosts=self.bad_hosts
                             )
-                        except NoPlatformsError:
+                        except PlatformLookupError:
```

Comment:
This case has only changed to make the error catching more general - the response to an error is unchanged.